packages/components/src/Page/Page.test.tsx

```diff
@@ -6,10 +6,12 @@ import { SectionProps } from "../Menu";
 
 jest.mock("@jobber/hooks", () => {
   return {
+    ...(jest.requireActual("@jobber/hooks") as {}),
```

Comment:
Will it be better to mock what you need rather than all of it?

Reply:
@darryltec, when mocking the entire `@jobber/hooks` module, if you don't require actual before what we actually want to mock, other hooks are no longer defined.

In this case, we're wanting to test `useOnKeydown` hook, and don't actually need to mock it, since it's quite easy to test, whereas `useResizeObserver` is hard to test and should be mocked.

In this case, we pull in everything that's defined, and override what should be mocked below.